iis-2vm-sql-1vm/azuredeploy.json

```diff
@@ -0,0 +1,637 @@
+        "description": "The type of the Storage Account created"
+      },
+      "defaultValue": "Premium_LRS"
+    }
+  },
+  "variables": {
+    "apiVersion": "2015-06-15",
+    "storageName": "[concat(parameters('envPrefixName'), 'stg', uniqueString(resourceGroup().id))]",
+    "vhdContainer": "vhds",
+    "virtualNetworkName": "[concat(parameters('envPrefixName'), 'Vnet')]",
+    "addressPrefix": "10.0.0.0/16",
+    "feSubnetPrefix": "10.0.0.0/24",
+    "dbSubnetPrefix": "10.0.2.0/24",
+    "feNSGName": "feNsg",
+    "dbNSGName": "dbNsg",
+    "sqlSrvDBName": "[concat(parameters('envPrefixName'), 'sqlSrv14')]",
+    "sqlVmSize": "[parameters('sqlVMSize')]",
+    "sqlSrvDBOsDiskName": "[concat(parameters('envPrefixName'), 'DBSrvOS')]",
+    "sqlSrvDBDataDiskName": "[concat(parameters('envPrefixName'), 'DBSrvData')]",
+    "vnetId": "[resourceId('Microsoft.Network/virtualNetworks', variables('virtualNetworkName'))]",
+    "sqlSrvDBNicName": "[concat(variables('sqlSrvDBName'), 'Nic')]",
+    "sqlSvrDBSubnetRef": "[concat(variables('vnetId'), '/subnets/', 'DBSubnetName')]",
+    "sqlPublicIP": "[concat(parameters('envPrefixName'), 'SqlPip')]",
+    "sqlPublicIPRef": "[resourceId('Microsoft.Network/publicIPAddresses', variables('sqlPublicIP'))]",
+    "sqlImagePublisher": "MicrosoftSQLServer",
+    "sqlImageOffer": "SQL2014-WS2012R2",
+    "sqlImageSku": "Standard",
+    "webSrvName": "[concat(parameters('envPrefixName'), 'webSrv')]",
+    "webSrvVMSize": "[parameters('webSrvVMSize')]",
+    "webSrvOSDiks": "[concat(parameters('envPrefixName'), 'WebSrvOS')]",
+    "webSrvNicName": "[concat(variables('webSrvName'), 'Nic')]",
+    "webSrvSubnetRef": "[concat(variables('vnetId'), '/subnets/', 'FESubnetName')]",
+    "webSrvPublicIP": "[concat(parameters('envPrefixName'), 'websrvpip')]",
+    "webSrvAvailabilitySetName": "[concat(parameters('envPrefixName'), 'webSrvAS')]",
+    "webSrvNumbOfInstances": "[parameters('numberOfWebSrvs')]",
+    "webSrvDnsNameforLBIP": "[concat(toLower(variables('webSrvName')), 'lb')]",
+    "webLbName": "[concat(variables('webSrvName'), 'lb')]",
+    "webLblbID": "[resourceId('Microsoft.Network/loadBalancers',variables('webLbName'))]",
+    "webLblb_PIP_Id": "[resourceId('Microsoft.Network/publicIPAddresses',variables('webSrvPublicIP'))]",
+    "webLbId": "[resourceId('Microsoft.Network/loadBalancers',variables('webLbName'))]",
+    "frontEndIPConfigID": "[concat(variables('webLbId'),'/frontendIPConfigurations/LoadBalancerFrontEnd')]",
+    "lbPoolID": "[concat(variables('webLbId'),'/backendAddressPools/BackendPool1')]",
+    "lbProbeID": "[concat(variables('webLbId'),'/probes/tcpProbe')]",
+    "vmExtensionName": "dscExtension",
+    "modulesUrl": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/iis-2vm-sql-1vm/scripts/WebServerConfig.ps1.zip",
+    "configurationFunction": "WebServerConfig.ps1\\WebServerConfig"
+  },
+  "resources": [
+    {
+      "apiVersion": "2015-06-15", 
```

Comment:
I see you created a variable apiVersion above but it's not used here and for the other resources below